files/syncano-client-js/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "Eyedea"
+github = "https://github.com/syncano/syncano-client-js"
+homepage = "http://syncano.io/"
+description = "Interact with your Syncano Sockets."
+mainfile = ""lib/index.js","
```

Comment:
ah, it got confused because of the trailing comma https://github.com/jsdelivr/jsdelivr/pull/16491/commits/a6aa33f6b6d29f01147443aee2a863eb0f747986#diff-4e1aafe073f6c49e99ea9db935c2f257R5